### PR TITLE
React Capstone, Crypto tracker

### PR DESCRIPTION
**Pull request by getssh**
- Create react app with create-react-app
- Installed necessary dependencies
- Install react-router-dom, redux-toolkit, testing-library, etc.
- Set up application directory structure
- Write action, reducers
- Fetch data from API
- Rendered the fetched data in components
- used hooks like useEffect, useSelector and useDispatch
- Followed the given design 
- Write testing files
- Tested if the components render the fetch data correctly
- Used mock data for testing
- Checked all files for linter error
- Update the README file
- Deployed the application on Render
- Prepared and included loom presentation link in the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +136,7 @@ npm run build
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/getssh/bookstore/issues).
+Feel free to check the [issues page](https://github.com/getssh/crypto-tracker-app/issues).
```
Comment (codepantha) [inline README.md:139]: - [x] In the "Acknowledgement" section of your README, please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    (item) => item.name.toLowerCase().includes(query.toLowerCase()),
+  ) : null;
+
+  const handleChange = (event) => {
+    setQuery(event.target.value);
+  };
+
+  return (
+    <div>
+      <input className="py-1 pl-5 w-full border border-1 border-black" placeholder="Search" type="text" value={query} onChange={handleChange} />
```
Comment (codepantha) [inline src/components/Home.js]: - [x] According to the project requirement, please use a `Filter` stateless component to filter the coin data retrieved from the API

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      <div className="flex justify-around items-center bg-[#fd5294] text-white py-7">
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-14 h-30 text-[#555]">
+          <path strokeLinecap="round" strokeLinejoin="round" d="M12 21a9.004 9.004 0 008.716-6.747M12 21a9.004 9.004 0 01-8.716-6.747M12 21c2.485 0 4.5-4.03 4.5-9S14.485 3 12 3m0 18c-2.485 0-4.5-4.03-4.5-9S9.515 3 12 3m0 0a8.997 8.997 0 017.843 4.582M12 3a8.997 8.997 0 00-7.843 4.582m15.686 0A11.953 11.953 0 0112 10.5c-2.998 0-5.74-1.1-7.843-2.918m15.686 0A8.959 8.959 0 0121 12c0 .778-.099 1.533-.284 2.253m0 0A17.919 17.919 0 0112 16.5c-3.162 0-6.133-.815-8.716-2.247m0 0A9.015 9.015 0 013 12c0-1.605.42-3.113 1.157-4.418" />
+        </svg>
+        <p className="font-medium">TRENDING COINS</p>
+      </div>
+      <h3 className="bg-[#df4783] text-white pl-2">STATS BY TRENDING COIN</h3>
```
Comment (codepantha) [inline src/components/Home.js:30]: - [x] For the typography across your entire app, please follow the design guidelines in terms of font face, size and weight
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @getssh 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Responsive UI 📌 
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @getssh, 

To highlight 👍🏽 
✔️ Designer is acknowledged in README
✔️ Your app typography matches the given typography
✔️ A Filter stateless component filters data retrieved from the API.

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**